app/docs/0.12.x/loadbalancing.md

```diff
@@ -146,6 +160,9 @@ the initial setup only features 2 targets.
 The tradeoff here is that the higher the number of slots, the better the random 
 distribution, but the more expensive the changes are (add/removing targets)
 
+Detailed information on adding and manipulating
+upstreams is available in the `upstream` section of the [Admin API reference][upstream-object-reference].
```

Comment:
it seems like the below change is trying to limit the umber of columns (laudable!), but this one was left untouched :(